### PR TITLE
vSphere-cpi update docs for human readable names

### PR DESCRIPTION
**Pull request by neilcx**
The BOSH-vSphere-cpi team changed  human readable names feature to false/disabled by default.  Need a document update.

### CODE REVIEW

File: content/vsphere-human-readable-names.md [hunk 1]
```diff
@@ -22,14 +21,13 @@ With this feature enabled, when a new vm is created, it will be assigned to a na
...
 
 2. Deploy the Director
 
 
-# More about human readable names
-
+## More about human readable names
 1. Currently vSphere CPI only support names in ASCII characters.
 
 2. The total max length kept for instance group name and deployment name is 65. If original names are longer, names will be trimmed. A 25 digits space is  guaranteed for deployment name part if need to trim; for instance group name, the number is 40.
```
Comment (EleanorRigby) [inline content/vsphere-human-readable-names.md]: You can remove line `A 25 digits space...`  => This level of implementation detail is not necessary here.

Comment (neilcx) [reply content/vsphere-human-readable-names.md]: Done.

**Comment by EleanorRigby** [comment]
@neilcx All commits here are concerned with human-readable names. So, please squash all of your commits into one commit.

**Comment by neilcx** [comment]
@EleanorRigby  Done.

**Pull request merged**